app/routes/_auth.tsx

```diff
@@ -1,8 +1,8 @@
 import type { V2_MetaFunction } from '@remix-run/react';
 import { Outlet } from '@remix-run/react';
 
-export const meta: V2_MetaFunction = () => {
-  return [{ title: 'Auth | Ore+' }];
+export const meta: V2_MetaFunction = (value: any) => {
+  return [{ title: 'Login | Orem+' }];
```

Comment:
coisa boba, mas pra evitar esses returns, quando a funçao for simples, melhor usar o shorthand pra retornar com ( ), evita ficar escrevendo return e deixa a funçao one-line